.latexmkrc

```diff
@@ -0,0 +1,24 @@
+add_cus_dep( 'acn', 'acr', 0, 'run_makeglossaries' );
+add_cus_dep( 'glo', 'gls', 0, 'run_makeglossaries' );
+add_cus_dep( 'ntn', 'not', 0, 'run_makeglossaries' );
+
+push @generated_exts, 'glo', 'gls', 'glg';
+push @generated_exts, 'acn', 'acr', 'alg';
```

Comment:
I saw glo/gls files are still generated (due to the automorphism example), but are acr/acn files supposed to be generated too?

Reply:
No, but - just in case.